### PR TITLE
New "Team" section on About page

### PR DESCRIPTION
**Pull request by jaspermdegroot**
TO DO:
- [ ] Text Alex
- [x] Text Anne
- [ ] Text Gabriel
- [ ] Text Amanpreet
- [x] Text Cory
- [x] Text Jasper
- [x] Add `target="_blank"` to GitHub profile links
- [ ] Remove inline styles from pages/about.html: Open PR in jquery-wp-content repo to add styles to the jquerymobile.com stylesheet

Screenshot:
about jquery mobile

Small screens:
about jquery mobile small

### CODE REVIEW

**Comment by jaspermdegroot** [comment]
@arschmitz @agcolom @gabrielschulhof @cgack - Can you please paste a small text about yourself in a comment here?

**Comment by cgack** [comment]
Cory Gackenheimer is a software engineer at [Healthx](http://healthx.com) by day. After hours he works as a [writer](http://www.apress.com/9781430260585), speaker, and consultant. In his role with jQuery Mobile he works to triage bugs and implementing new features.

File: pages/about.html [hunk 1]
```diff
@@ -38,3 +26,92 @@
...
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/acabeff49b5ba95a8e94de9445175f56?size=120" alt="Avatar Gabriel Schulhof">
+        <h3><a href="https://github.com/gabrielschulhof" title="Gabriel Schulhof GitHub profile">Gabriel Schulhof</a></h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/691460b4005b0c16bcb96dcdf1f0e02d?size=120" alt="Avatar Cory Gackenheimer">
+        <h3><a href="https://github.com/cgack" title="Cory Gackenheime GitHub profile">Cory Gackenheimer</a></h3>
```
Comment (cgack) [inline pages/about.html]: Missing the last letter of my last name in the `title` attribute in the link to my github profile

File: pages/about.html [hunk 2]
```diff
@@ -38,3 +26,92 @@
...
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/e8170d7574cd649bce2639e24737de88?size=120" alt="Avatar Jasper de Groot">
+        <h3><a href="https://github.com/jaspermdegroot" title="Jasper de Groot GitHub profile">Jasper de Groot</a></h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+</ul>
+
+<p>The Mobile team is having its weekly meeting at Thursday 2 PM (EST) on <a href="http://irc.jquery.org">IRC</a> in the #jquery-meeting channel. Meeting notes are posted on http://meetings.jquery.org/category/mobile/ after each meeting.</p>
```
Comment (jzaefferer) [inline pages/about.html]: Make http://meetings.jquery.org/category/mobile/ a link.

**Comment by agcolom** [comment]
@jaspermdegroot Just noticed the url for meeting notes at the bottom is http://vagrant.?
Shouldn't it just be http://meetings.jquery.org/category/mobile/ ?

**Comment by jaspermdegroot** [comment]
@agcolom - That's only in your local environment. In the commit itself you'll see the correct url for the live site.

**Comment by agcolom** [comment]
@jaspermdegroot Oh, sorry, I looked at the screenshot ;-)

**Comment by rgfpy** [comment]
force to revive the project JQM

File: pages/about.html [hunk 3]
```diff
@@ -38,3 +26,92 @@
...
+    <li>
+        <img src="https://secure.gravatar.com/avatar/691460b4005b0c16bcb96dcdf1f0e02d?size=120" alt="Avatar Cory Gackenheimer">
+        <h3><a href="https://github.com/cgack" title="Cory Gackenheimer GitHub profile" target="_blank">Cory Gackenheimer</a></h3>
+        <p>Cory Gackenheimer is a software engineer at <a href="http://www.healthx.com/" target="_blank">Healthx</a> by day. After hours he works as a <a href="http://www.apress.com/9781430260585" title="Apress" target="_blank">writer</a>, speaker, and consultant. In his role with jQuery Mobile he works to triage bugs and implementing new features.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/e8170d7574cd649bce2639e24737de88?size=120" alt="Avatar Jasper de Groot">
+        <h3><a href="https://github.com/jaspermdegroot" title="Jasper de Groot GitHub profile" target="_blank">Jasper de Groot</a></h3>
+        <p>Jasper de Groot is a Dutch freelance web developer, residing in Barcelona, Spain. He is specializing in WordPress theme development. Jasper joined the team in 2012 and focuses on the CSS side of the project and the demos.</p>
+    </li>
```
Comment (arthurvr) [inline pages/about.html]: This indentation seems to be wrong. (It's 4 spaces whilest it should be tabs)

**Comment by jaspermdegroot** [comment]
@arschmitz @gabrielschulhof bump!

**Comment by jaspermdegroot** [comment]
@arschmitz @gabrielschulhof bump!!

**Comment by jaspermdegroot** [comment]
@arschmitz @gabrielschulhof 

Any chance you two can post a small text about yourself in a comment here? Come on guys, I keep bumping. Things like this really shouldn't take 10 months!

**Comment by mgol** [comment]
This is obsolete now, closing.

**Pull request closed without merging**